src/test/java/com/microsoft/azure/spring/data/cosmosdb/domain/MemoType.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.data.cosmosdb.domain;
+
+public enum MemoType {
+    HAPPY("happy"),
```

Comment:
What does `happy` and `sad` really mean? Maybe more accurate.

Reply:
It means a `happy` `MemoType` and a `sad` `MemoType`.

Reply:
It's still unclear for me what's happy.

Reply:
As this is different enum types for memoType, which means it's happy memory type and sad memory type etc., do not think it's confusing or unclear though, also the point here is to test enum but not for any production.

Reply:
Or you can give any suggestion which is clear to you.